package.json

```diff
@@ -9,7 +9,7 @@
     "@babel/node": "^7.13.0",
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/preset-env": "^7.13.5",
-    "@hathor/wallet-lib": "^0.23.0",
+    "@hathor/wallet-lib": "^0.25.0",
```

Comment:
We should wait for the new version of wallet-lib, including a fix for a bug in 0.25.0